pkgs/by-name/ap/apriltag/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  opencv4,
+  python3,
+}:
+
+let
+  opencv4WithGtk = opencv4.override {
+    enableGtk2 = true; # For GTK2 support
+    enableGtk3 = true; # For GTK3 support
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "AprilTags";
+  version = "3.4.3";
+
+  src = fetchFromGitHub {
+    owner = "AprilRobotics";
+    repo = "AprilTags";
+    tag = "v${version}";
+    sha256 = "sha256-1XbsyyadUvBZSpIc9KPGiTcp+3G7YqHepWoORob01Ss=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    python3
+  ];
+
+  buildInputs = [ opencv4WithGtk ];
+
+  propagatedBuildInputs = [ python3.pkgs.numpy ];
+
+  cmakeFlags = [ "-DBUILD_EXAMPLES=ON" ];
+
+  doCheck = true;
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
Nit: [avoid wide `with` scopes](https://nix.dev/guides/best-practices#with-scopes).

---

How do I access then the `maintainers`?

```
       error: syntax error, unexpected '['
       at /home/pethod/nixpkgs/pkgs/by-name/ap/apriltag/package.nix:44:32:
           43|     platforms = platforms.all;
           44|     maintainers = maintainers; [ phodina ];
             |                                ^
           45|   };

```

---

`maintainers = with lib.maintainers; [ phodina ]`